src/global.css

```diff
@@ -0,0 +1,56 @@
+@import-normalize; /* bring in normalize.css styles */
```

Comment:
You can use styled-components' [createGlobalStyle](https://styled-components.com/docs/api#createglobalstyle) instead of global.css